### PR TITLE
certificate decode error when decoding from bash pipe by openssl

### PR DESCRIPTION
**Pull request by SerenaFeng**
Current implementation can not decode the certificate but generate the following error:

```
$ kubectl get secret dns.istio-galley-service-account -n istio-system -o json | jq -r '.data["cert-chain.pem"]' | base64 --decode | openssl x509 -in - -text -noout
Error opening Certificate -
140406230840984:error:02001002:system library:fopen:No such file or directory:bss_file.c:398:fopen('-','r')
140406230840984:error:20074002:BIO routines:FILE_CTRL:system lib:bss_file.c:400:
unable to load certificate
```

### CODE REVIEW

File: content/en/docs/tasks/security/dns-cert/index.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ and that the certificate contains the configured DNS names, you need to get the
 decode it, and view its text output with the following command:
 
 {{< text bash >}}
-$ kubectl get secret dns.istio-galley-service-account -n istio-system -o json | jq -r '.data["cert-chain.pem"]' | base64 --decode | openssl x509 -in - -text -noout
+$ kubectl get secret dns.istio-galley-service-account -n istio-system -o json | jq -r '.data["cert-chain.pem"]' | base64 --decode | openssl x509 -in /dev/stdin -text -noout
```
Comment (frankbu) [inline content/en/docs/tasks/security/dns-cert/index.md]: Isn't this same change in your other PR? #6245

Comment (SerenaFeng) [reply content/en/docs/tasks/security/dns-cert/index.md]: This change is mistakenly mingled with #6245 , I have submitted a new pull request (#6282 ) to substitute #6245 .

Comment (frankbu) [reply content/en/docs/tasks/security/dns-cert/index.md]: @SerenaFeng I'm still not sure I understand why you can't use `-` for the `-in` argument, but others can. What platform are you using? Are you using a different version of `openssl` than others? If so, are you sure your fix works on all version?

Comment (frankbu) [reply content/en/docs/tasks/security/dns-cert/index.md]: @lei-tang can you confirm that `-in -` works for you? What platform and version of openssl are you using?

Comment (lei-tang) [reply content/en/docs/tasks/security/dns-cert/index.md]: "-in -" works for me. In my environment, the OS is debian and the openssl version is: OpenSSL 1.1.1d  10 Sep 2019.

Comment (SerenaFeng) [reply content/en/docs/tasks/security/dns-cert/index.md]: In my environment, `-in /dev/stdin` works on`OpenSSL 1.0.2g  1 Mar 2016` ubuntu xenial and `OpenSSL 0.9.8zh 14 Jan 2016` macOS Sierra v10.12.6, while `-in -` not. 

For `OpenSSL 1.1.1d  10 Sep 2019` ubuntu xenial situation, both `-in /dev/stdin` and `-in -` work correctly.

Comment (frankbu) [reply content/en/docs/tasks/security/dns-cert/index.md]: So it sounds like OpenSSL 1.1 supports `-in -`, but older versions do not. We could fix this either by changing to `-in /dev/stdin` or by specifying that OpenSLL >= 1.1 is needed for this task.

@lei-tang You added this task, so WDYT?

Comment (lei-tang) [reply content/en/docs/tasks/security/dns-cert/index.md]: As "-in /dev/stdin" works for both older and newer versions of openssl, I think we can changing the command to use "-in /dev/stdin".

**Pull request merged**